### DISCUSSION TITLE
为什么 incubator-doris-flink-connector 没有定义「sink.batch.bytes」表属性

### DISCUSSION THREAD

**Discussion by banposeu** [apache/doris#8608] [A - General / Q&A] [answered]
flink connector doc 文档中配置属性中有「sink.batch.bytes」属性，但是在 incubator-doris-flink-connector 却没有定义，请问是预留属性嘛，还是代码有遗漏？
https://github.com/apache/incubator-doris/blob/master/docs/zh-CN/extending-doris/flink-doris-connector.md

**Comment by hf200012** [comment, marked as answer]
> flink connector doc 文档中配置属性中有「sink.batch.bytes」属性，但是在 incubator-doris-flink-connector 却没有定义，请问是预留属性嘛，还是代码有遗漏？
> https://github.com/apache/incubator-doris/blob/master/docs/zh-CN/extending-doris/flink-doris-connector.md

PR刚合并，你可以拉取最新代码，编译
